src/status_im/contexts/chat/group_details/view.cljs

```diff
@@ -50,7 +50,7 @@
            item])))))
 
 (defn add-manage-members
-  [{:keys [scroll-enabled? on-scroll]}]
```

Comment:
but we can't just remove these props? they are needed

Reply:
@flexsurfer do you know why we need them. I dont see why we need them on this part and scroll-enabled? is the cause for this issue.

Reply:
it is used to close the screen by scrolling, but still we need to know the reason, we can't just remove functionality

Reply:
@flexsurfer  the screen here will always re-render when `scroll-enabled` is updated and this is why scrolling stops working for a moment because its actually disabled. 

I am not sure but assuming this is reason why this enabling and disabling of scroll was avoided in other areas like `invite contacts to community` where its the same list of contacts and the same functionality would probably be needed